packages/editor-ui/src/views/NodeView.v2.vue

```diff
@@ -1385,6 +1386,22 @@ function registerCustomActions() {
 	});
 }
 
+/**
+ * Unload
+ */
+
+function onBeforeUnload(e: BeforeUnloadEvent) {
+	if (isDemoRoute.value || window.preventNodeViewBeforeUnload) {
```

Comment:
What's this `preventNodeViewBeforeUnload` flag? I don't see it being set anywhere in the code

Reply:
it's used in the e2e tests to skip this prompt